Point_set_processing_3/include/CGAL/constraint_based_smooth_point_set.h

```diff
@@ -0,0 +1,652 @@
+// Copyright (c) 2023 GeometryFactory (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s) : 
+
+#ifndef CGAL_CONSTRAINT_BASED_SMOOTH_POINT_SET_H
+#define CGAL_CONSTRAINT_BASED_SMOOTH_POINT_SET_H
+
+#include <CGAL/license/Point_set_processing_3.h>
+
+#include <cmath>
+#include <CGAL/disable_warnings.h>
+
+#include <CGAL/Point_set_processing_3/internal/Neighbor_query.h>
+#include <CGAL/compute_average_spacing.h>
+#include <CGAL/Min_sphere_of_points_d_traits_3.h>
+#include <CGAL/Min_sphere_of_spheres_d.h>
+// #include <CGAL/Point_set_processing_3/internal/Callback_wrapper.h>
+#include <CGAL/for_each.h>
+
+#include <CGAL/Named_function_parameters.h>
+#include <CGAL/boost/graph/named_params_helper.h>
+
+#include <boost/iterator/zip_iterator.hpp>
+
+#include <CGAL/Eigen_vector.h>
+#include <CGAL/Eigen_matrix.h>
+#include <CGAL/Kernel/global_functions.h>
+
+namespace CGAL {
+
+
+// ----------------------------------------------------------------------------
+// Private section
+// ----------------------------------------------------------------------------
+/// \cond SKIP_IN_MANUAL
+
+namespace internal {
```

Comment:
As you can see above in line 20, the namespace `internal` is inside `CGAL::Point_set_processing`.